include/cec/CECHandler.h

```diff
@@ -0,0 +1,65 @@
+#pragma once
+
+#include <QObject>
+#include <QVector>
+
+#include <iostream>
+
+#include <libcec/cec.h>
+
+#include <cec/CECEvent.h>
+
+using CECCallbacks         = CEC::ICECCallbacks;
+using CECAdapter           = CEC::ICECAdapter;
+using CECAdapterDescriptor = CEC::cec_adapter_descriptor;
+using CECLogMessage        = CEC::cec_log_message;
+using CECKeyPress          = CEC::cec_keypress;
+using CECCommand           = CEC::cec_command;
+using CECLogicalAddress    = CEC::cec_logical_address;
+using CECLogicalAddresses  = CEC::cec_logical_addresses;
+using CECMenuState         = CEC::cec_menu_state;
+using CECPowerStatus       = CEC::cec_power_status;
+using CECVendorId          = CEC::cec_vendor_id;
+using CECParameter         = CEC::libcec_parameter;
+using CECConfig            = CEC::libcec_configuration;
+using CECAlert             = CEC::libcec_alert;
+
```

Comment:
I don't know why the whole file looks like I used spaces instead of tabs whereas I triplechecked that I used tabs.